rollup.config.js

```diff
@@ -28,7 +28,37 @@ const plugins = (es5) => [
             postcssImport(),
             postcssNesting(),
             cssnano({
-                preset: ['default', { discardComments: { removeAll: true } }],
+                preset: [
```

Comment:
feel free to look past this, main goal of this is to reduce our CSS files as much as possible